test/testdrive/hydration-status.td

```diff
@@ -56,16 +55,16 @@ s true
 > CREATE MATERIALIZED VIEW mv AS SELECT * FROM t
 > CREATE MATERIALIZED VIEW mv_const AS SELECT 1
 
-> SELECT o.name, r.name, h.hydrated
+> SELECT o.name, r.name, h.hydrated, h.hydration_time < '5s'
```

Comment:
Isn't this potentially flaky?

Reply:
I figured 5s should be safe and if it flakes we'd want to look into it. But yes, in theory it's flaky. @MaterializeInc/testing would you prefer we change this to `h.hydration_time IS NOT NULL` instead?

Reply:
I definitely prefer non-flaky in our tests. CI seems to be great at causing flakes. Would we do something other than bump the hydration time if there is a flake here?

Reply:
Yeah, probably nothing else than be sad that things are sometimes that slow :/ I'll change it.